aws/logs_monitoring/tests/approved_files/TestLambdaFunctionEndToEnd.test_datadog_forwarder.approved.json

```diff
@@ -11,14 +11,14 @@
         },
         "ddsource": "lambda",
         "ddsourcecategory": "aws",
-        "ddtags": "forwardername:inferred-spans-python-dev-initsender,forwarder_memorysize:10,forwarder_version:<redacted from snapshot>,env:none,account_id:601427279990,aws_account:601427279990,functionname:hello-dog-node-dev-hello12x,region:sa-east-1,service:hello-dog-node-dev-hello12x",
+        "ddtags": "forwardername:inferred-spans-python-dev-initsender,forwarder_memorysize:10,forwarder_version:<redacted from snapshot>,,account_id:601427279990,aws_account:601427279990,creator:swf,env:prod,functionname:hello-dog-node-dev-hello12x,monitor:datadog,region:sa-east-1,service:hello,team:metrics",
```

Comment:
are all the changes in this file on purpose ? fixing bugs ?

Reply:
Yes, the behavior of this test should set the tags from the lambda function tags cache which we're mocking. The previous approved result was not matching the expected behavior